atomic_reactor/plugins/post_pulp_sync.py

```diff
@@ -208,18 +208,22 @@ def run(self):
             if image.pulp_repo not in repos:
                 repo_id = self.create_repo_if_missing(pulp,
                                                       image.pulp_repo,
-                                                      image.repo)
+                                                      image.to_str(registry=False, tag=False))
```

Comment:
A test to ensure `createRepo` is called with correct args and only at expected scenarios would be a great improvement on the existing tests.